backend/src/apiserver/resource/resource_manager.go

```diff
@@ -875,7 +891,7 @@ func (r *ResourceManager) ReportWorkflowResource(ctx context.Context, workflow *
 				StorageState:     api.Run_STORAGESTATE_AVAILABLE.String(),
 				Namespace:        workflow.Namespace,
 				CreatedAtInSec:   workflow.CreationTimestamp.Unix(),
-				ScheduledAtInSec: workflow.ScheduledAtInSecOr0(),
+				ScheduledAtInSec: scheduledTimeInSec,
```

Comment:
Maybe we can just return `workflow.ScheduledAtInSecOr0()` here, because frontend should handle the `value equals to zero` case anyway.

Reply:
See the discussion about definition of "ScheduledAt".